patches/net/minecraft/world/level/Level.java.patch

```diff
@@ -235,6 +235,11 @@
 +        if (value > maxEntityRadius)
 +            maxEntityRadius = value;
 +        return maxEntityRadius;
++    }
++
++    @Override
++    public net.neoforged.neoforge.client.model.data.ModelDataManager.Active getModelDataManager() {
```

Comment:
Technically this should be annotated with nullable, else you are saying that the method does not return null. (Since `T` is a subtype of `@Nullable T`).